### PR TITLE
feat(cli): add awf init local bootstrap command

### PR DESCRIPTION
**Pull request by dimileeh**
Automatically opened by AWF workspace `ws_0da5b57348cb49d198db9ee2` (agent: `claude_code`, model: `claude-opus-4-7`, effort: `xhigh`).

**External task ID**: core-readiness-p1-awf-init-local-bootstrap

### Task
You are working in the AWF repository. Follow AGENTS.md strictly: use TDD, keep the change narrowly scoped, preserve or improve coverage, and prefer existing src/awf patterns. The target branch is codex/awf-post-merge-fixes and auto_merge is enabled because this is not main. Do not manually merge the PR. Keep changes inside the declared owned_paths unless a test proves a tiny adjacent edit is required, and explain any such edit in the PR.

Validation expectation: run the narrow tests/lint/typecheck for touched areas, and run broader coverage when the change touches core behavior.

Backlog slice: P1 MCP And Project Onboarding Client Parity - one-command local bootstrap via `awf init`.

Goal: make local first-run setup absurdly easy for a new engineer.

Acceptance criteria:
- Add an `awf init` command or improve the existing top-level command so it checks Docker availability, ensures/prints local AWF state directory guidance, validates or writes safe local defaults when appropriate, starts or validates the local Postgres/API/worker/console stack through existing bootstrap mechanisms, and prints concise next steps.
- Keep existing `awf project/profile init`, `awf service bootstrap`, and `awf doctor/status` behavior compatible.
- Add focused unit tests around CLI parsing/output and the bootstrap orchestration boundaries; mock shell/service operations rather than starting Docker in unit tests.
- Update README/docs with the recommended install/bootstrap path, while keeping git clone as contributor/development setup.

---
Validation: 5 profile command(s) passed inside the workspace container.

### CODE REVIEW

File: src/awf/cli/main.py [hunk 1]
```diff
@@ -368,6 +458,150 @@ async def _collect_cached_service_status(
...
+            _emit(
+                {
+                    "status": "failed",
+                    "reason_code": reason,
+                    "message": message,
+                    "action": action,
+                },
+                fmt,
+            )
+        raise typer.Exit(code=1)
```
[the remark this answers is not in the document]
Comment (dimileeh) [reply src/awf/cli/main.py]: fixed in commit 39af05cd56330e763128c231a4a4694746ebac32

File: src/awf/cli/main.py [hunk 2]
```diff
@@ -368,6 +458,150 @@ async def _collect_cached_service_status(
...
+    docker_report = asyncio.run(
+        collect_doctor_report(
+            settings,
+            strict_providers=frozenset(),
+            provider_environ=os.environ,
+            environ=os.environ,
+        )
+    )
+    docker_diag = _docker_diagnostic_from_report(docker_report)
+    if docker_diag is not None and getattr(docker_diag, "status", "ok") == "fail":
```
[the remark this answers is not in the document]
Comment (dimileeh) [reply src/awf/cli/main.py]: fixed in commit 47ca0fc4d10ffaee96073f4293819ef249519a3c

File: src/awf/cli/main.py [hunk 3]
```diff
@@ -368,6 +458,150 @@ async def _collect_cached_service_status(
...
+
+    pretty = fmt == OutputFormat.pretty
+    settings = resolve_service_settings()
+
+    if pretty:
+        typer.echo("AWF init: local service bootstrap")
+
+    docker_report = asyncio.run(
+        collect_doctor_report(
+            settings,
```
[the remark this answers is not in the document]
Comment (dimileeh) [reply src/awf/cli/main.py]: fixed in commit c9baeb726dde81023bd9a010471d1d690c0d141d

File: src/awf/cli/main.py [hunk 4]
```diff
@@ -237,16 +237,106 @@ def _handle_response(
...
+            f"{', '.join(bootstrap_only_flags)} require running `awf init` "
+            "without a project path.",
+            err=True,
+        )
+        raise typer.Exit(code=2)
+
+    _run_init_project_onboarding(
+        path,
+        include_smoke_request=include_smoke_request,
+    )
```
[the remark this answers is not in the document]
Comment (dimileeh) [reply src/awf/cli/main.py:350]: fixed in commit b8f22451b9c561487349dedea9ae509aa3bbee35

File: src/awf/cli/main.py [hunk 5]
```diff
@@ -237,16 +237,106 @@ def _handle_response(
...
+    if path is None:
+        _run_init_service_bootstrap(
+            write_env=write_env,
+            timeout_seconds=timeout_seconds,
+            poll_interval_seconds=poll_interval_seconds,
+            skip_agent_runtime_build=skip_agent_runtime_build,
+            providers=provider,
+            fmt=fmt,
+        )
+        return
```
[the remark this answers is not in the document]
Comment (dimileeh) [reply src/awf/cli/main.py:318]: fixed in commit 89d82dbcd6353e1781326ba862c85e5cf09dc288

File: src/awf/cli/main.py [hunk 6]
```diff
@@ -368,6 +458,150 @@ async def _collect_cached_service_status(
...
+        typer.echo("AWF init: local service bootstrap")
+
+    docker_report = asyncio.run(
+        collect_doctor_report(
+            settings,
+            strict_providers=frozenset(),
+            provider_environ=os.environ,
+            environ=os.environ,
+        )
+    )
```
[the remark this answers is not in the document]
Comment (dimileeh) [reply src/awf/cli/main.py]: FALSE POSITIVE: The two `asyncio.run` calls are independent operations separated by substantial synchronous logic (a Docker-availability check that may early-exit, state-directory creation, and `.env` writing). The underlying coroutines (`collect_doctor_report` and `run_service_bootstrap`) hold no shared async resources and do not rely on a persistent event loop — `run_service_bootstrap` only uses `asyncio.to_thread` and `asyncio.sleep` internally, which work within any event loop. Wrapping both phases in a single async function would force the intermediate synchronous I/O (filesystem checks, `.env` write) into the async context without any functional benefit; the existing `_collect_reports` pattern at line 425 in the same file uses one `asyncio.run` because it needs `asyncio.gather` for concurrency, which is not the case here.

File: src/awf/cli/main.py [hunk 7]
```diff
@@ -368,6 +458,150 @@ async def _collect_cached_service_status(
...
+    created = not state_dir.exists()
+    state_dir.mkdir(parents=True, exist_ok=True)
+    if pretty:
+        typer.echo(f"  state directory: {state_dir}")
+        typer.echo(f"  created: {'true' if created else 'false'}")
+
+    env_action = "skipped"
+    if write_env:
+        env_file = Path(".env")
+        env_example = Path(".env.example")
```
[the remark this answers is not in the document]
Comment (dimileeh) [reply src/awf/cli/main.py:597]: fixed in commit c949b325c87249b04a35004911ded3b84e70af28

File: src/awf/cli/main.py [hunk 8]
```diff
@@ -237,16 +237,117 @@ def _handle_response(
...
+        )
+        return
+
+    bootstrap_only_flags: list[str] = []
+    if skip_agent_runtime_build:
+        bootstrap_only_flags.append("--skip-agent-runtime-build")
+    if provider:
+        bootstrap_only_flags.append("--provider")
+    if timeout_seconds != _DEFAULT_INIT_BOOTSTRAP_TIMEOUT_SECONDS:
+        bootstrap_only_flags.append("--timeout-seconds")
```
[the remark this answers is not in the document]
Comment (dimileeh) [reply src/awf/cli/main.py]: fixed in commit f5e4a45e032ee4fe12e93c3f6ce91e33c0f3dac7
